Started UtBot Contest, classfileDir = /var/benchmarks/projects/guava/guava/target/classes, classpathString=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, outputDir=/home/utbot_mocks/temp/testcases, mocks=Other classes: Mockito
19:03:12.531 | INFO  | Started: warmup: kotlin reflection :: init
19:03:12.604 | INFO  | Finished (in 0.072 sec): warmup: kotlin reflection :: init 
[INIT]
>> [RUN] com.google.common.base.internal.Finalizer 30
19:03:12.615 | DEBUG | -----------------------------------------------------------------------------
19:03:12.615 | INFO  | Contest.runGeneration: Time budget: 30000 ms, Generation timeout=25500 ms, classpath=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, methodNameFilter=null
19:03:12.616 | INFO  | ClassUnderTest[ FQN: com.google.common.base.internal.Finalizer
    classfileDir: /var/benchmarks/projects/guava/guava/target/classes
    testClassSimpleName: FinalizerTest
    generatedTestFile: /home/utbot_mocks/temp/testcases/com/google/common/base/internal/FinalizerTest.java
    generatedTestsSourcesDir: /home/utbot_mocks/temp/testcases
]
19:03:12.638 | INFO  | Started: class com.google.common.base.internal.Finalizer
19:03:12.638 | INFO  | Started: preparation class class com.google.common.base.internal.Finalizer: kotlin reflection :: run
19:03:12.663 | INFO  | Finished (in 0.024 sec): preparation class class com.google.common.base.internal.Finalizer: kotlin reflection :: run 
19:03:12.664 | INFO  | Started: 2nd optional soot initialization
19:03:12.665 | INFO  | Finished (in 0.001 sec): 2nd optional soot initialization 
19:03:12.675 | DEBUG |  ... 
19:03:12.686 | INFO  | Started: method com.google.common.base.internal.Finalizer.run()
19:03:12.689 | INFO  |  -- Remaining time budget: 25439 ms, #remaining_methods: 7, budget for method: 3634 ms, solver timeout: 1000 ms, budget for symbolic execution: 2434 ms, budget for concrete execution: 1200 ms,  -- 
19:03:13.006 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()1
19:03:14.344 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()2
19:03:15.119 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.run()
19:03:15.248 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.run()3
19:03:16.321 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.run()
19:03:16.367 | INFO  | Finished (in 3.678 sec): method com.google.common.base.internal.Finalizer.run() 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
19:03:16.418 | DEBUG |  ... 
19:03:16.419 | INFO  | Started: method com.google.common.base.internal.Finalizer.cleanUp(Reference)
19:03:16.419 | INFO  |  -- Remaining time budget: 21698 ms, #remaining_methods: 6, budget for method: 3616 ms, solver timeout: 1000 ms, budget for symbolic execution: 2416 ms, budget for concrete execution: 1200 ms,  -- 
19:03:16.431 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)1
19:03:16.908 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)2
19:03:17.060 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.cleanUp(Reference)3
19:03:18.835 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.cleanUp(Reference)
19:03:20.036 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.cleanUp(Reference)
19:03:20.061 | INFO  | Finished (in 3.641 sec): method com.google.common.base.internal.Finalizer.cleanUp(Reference) 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
19:03:20.163 | DEBUG |  ... 
19:03:20.163 | INFO  | Started: method com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
19:03:20.164 | INFO  |  -- Remaining time budget: 17953 ms, #remaining_methods: 5, budget for method: 3590 ms, solver timeout: 1000 ms, budget for symbolic execution: 2390 ms, budget for concrete execution: 1200 ms,  -- 
19:03:20.171 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()1
19:03:20.413 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()2
19:03:20.487 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getFinalizeReferentMethod()3
19:03:22.554 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
19:03:23.755 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getFinalizeReferentMethod()
19:03:23.780 | INFO  | Finished (in 3.616 sec): method com.google.common.base.internal.Finalizer.getFinalizeReferentMethod() 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
19:03:23.888 | DEBUG |  ... 
19:03:23.889 | INFO  | Started: method com.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference)
19:03:23.889 | INFO  |  -- Remaining time budget: 14227 ms, #remaining_methods: 4, budget for method: 3556 ms, solver timeout: 1000 ms, budget for symbolic execution: 2356 ms, budget for concrete execution: 1200 ms,  -- 
19:03:23.906 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
19:03:24.034 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference)1
19:03:24.075 | INFO  | Finished (in 0.186 sec): method com.google.common.base.internal.Finalizer.startFinalizer(Class, ReferenceQueue, PhantomReference) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
19:03:24.087 | DEBUG |  ... 
19:03:24.088 | INFO  | Started: method com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
19:03:24.088 | INFO  |  -- Remaining time budget: 14029 ms, #remaining_methods: 3, budget for method: 4676 ms, solver timeout: 1000 ms, budget for symbolic execution: 3476 ms, budget for concrete execution: 1200 ms,  -- 
19:03:24.107 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()1
19:03:26.855 | ERROR | Symbolic execution FAILED
java.lang.RuntimeException: No method source set for method <java.lang.String: java.lang.String intern()>
	at soot.SootMethod.retrieveActiveBody(SootMethod.java:398) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.ExtensionsKt.jimpleBody(Extensions.kt:136) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.BaseOverriddenWrapper.invoke(CollectionWrappers.kt:89) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.overrideInvocation(UtBotSymbolicEngine.kt:2663) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.commonInvokePart(UtBotSymbolicEngine.kt:2399) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.virtualAndInterfaceInvoke(UtBotSymbolicEngine.kt:2289) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.invokeResult(UtBotSymbolicEngine.kt:2153) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.traverseAssignStmt(UtBotSymbolicEngine.kt:884) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine.traverseStmt(UtBotSymbolicEngine.kt:604) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine$traverse$1.invokeSuspend(UtBotSymbolicEngine.kt:464) [utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.engine.UtBotSymbolicEngine$traverse$1.invoke(UtBotSymbolicEngine.kt) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:212) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__MergeKt$flattenConcat$$inlined$unsafeFlow$1$lambda$1.emit(Collect.kt:134) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__BuildersKt$flowOf$$inlined$unsafeFlow$1.collect(SafeCollector.common.kt:114) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.flow.FlowKt__BuildersKt$flowOf$$inlined$unsafeFlow$1$1.invokeSuspend(Unknown Source) [utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
19:03:27.246 | ERROR | Symbolic execution FAILED ... <<stack trace duplicated>>
19:03:27.564 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
19:03:28.765 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField()
19:03:28.838 | INFO  | Finished (in 4.750 sec): method com.google.common.base.internal.Finalizer.getInheritableThreadLocalsField() 
<StatsForMethod> :
	#generatedTC=1
	FAILED 2 time(s) with 1 different exception(s)"
19:03:28.957 | DEBUG |  ... 
19:03:28.958 | INFO  | Started: method com.google.common.base.internal.Finalizer.getBigThreadConstructor()
19:03:28.958 | INFO  |  -- Remaining time budget: 9158 ms, #remaining_methods: 2, budget for method: 4579 ms, solver timeout: 1000 ms, budget for symbolic execution: 3379 ms, budget for concrete execution: 1200 ms,  -- 
19:03:28.965 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getBigThreadConstructor()1
19:03:32.337 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.internal.Finalizer.getBigThreadConstructor()
19:03:33.008 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.getBigThreadConstructor()2
19:03:33.538 | INFO  | (X) Cancelling concrete execution: com.google.common.base.internal.Finalizer.getBigThreadConstructor()
19:03:33.593 | INFO  | Finished (in 4.635 sec): method com.google.common.base.internal.Finalizer.getBigThreadConstructor() 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
19:03:33.768 | DEBUG |  ... 
19:03:33.768 | INFO  | Started: method com.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference)
19:03:33.769 | INFO  |  -- Remaining time budget: 4348 ms, #remaining_methods: 1, budget for method: 4348 ms, solver timeout: 1000 ms, budget for symbolic execution: 3148 ms, budget for concrete execution: 1200 ms,  -- 
19:03:33.775 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
19:03:34.068 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference)1
19:03:34.069 | INFO  | Finished (in 0.300 sec): method com.google.common.base.internal.Finalizer.<init>(Class, ReferenceQueue, PhantomReference) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
19:03:34.083 | INFO  | Started: Flushing tests for [Finalizer] on disk
19:03:34.304 | INFO  | File size for FinalizerTest: 29 KB
19:03:34.306 | INFO  | Finished (in 0.223 sec): Flushing tests for [Finalizer] on disk 
19:03:34.307 | INFO  | Finished (in 21.669 sec): class com.google.common.base.internal.Finalizer 
<StatsForClass> :
	canceled by timeout = false
	#methods = 7, 
	#methods started symbolic exploration = 7
	#methods with at least one TC = 7
	#methods with exceptions = 3
	#generated TC = 14
	#coverage = null
[READY]
